src/test/java/redis/clients/jedis/tests/commands/ObjectCommandsTest.java

```diff
@@ -1,15 +1,41 @@
 package redis.clients.jedis.tests.commands;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
 
+import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
 
+import redis.clients.jedis.HostAndPort;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.tests.HostAndPortUtil;
 import redis.clients.jedis.util.SafeEncoder;
 
+import java.util.List;
+
 public class ObjectCommandsTest extends JedisCommandTestBase {
 
   private String key = "mylist";
   private byte[] binaryKey = SafeEncoder.encode(key);
+  private static final HostAndPort lfuHnp = HostAndPortUtil.getRedisServers().get(8);
```

Comment:
If you could do `HostAndPortUtil.getRedisServers().get(7)`, you won't need new Redis instance.

Reply:
So I remove the new Redis Instance?